arcade/scene.py

```diff
@@ -70,14 +67,11 @@ def __getitem__(self, key: str) -> SpriteList:
     def add_sprite(self, name: str, sprite: Sprite) -> None:
         """
         Add a Sprite to a SpriteList in the Scene with the specified name.
-
         If the desired SpriteList does not exist, it will automatically be created
         and added to the Scene. This will default the SpriteList to be added to the end
         of the draw order, and created with no extra options like using spatial hashing.
-
         If you need more control over where the SpriteList goes or need it to use Spatial Hash,
         then the SpriteList should be added separately and then have the Sprites added.
-
```

Comment:
Newline is needed here. I will not comment on all the other instances. You just need to revert all the changes you did to docstrings.